src/devices/gledopto.ts

```diff
@@ -1034,4 +1035,237 @@ export const definitions: DefinitionWithExtend[] = [
+    'Late autumn': [0x01, 0x1e, 0x0a, 0x52, 0x52, 0xe0, 0x00, 0x00, 0x64, 0x00, 0x19, 0x64, 0x00, 0x22, 0x5e, 0x00, 0x2c, 0x5b, 0x00, 0x14, 0x64, 0x00, 0x0c, 0x64],
+    'Game': [0x01, 0x1f, 0x02, 0x5f, 0x5f, 0x60, 0x00, 0x00, 0x64, 0x01, 0x10, 0x64, 0x00, 0xd2, 0x64, 0x00, 0xad, 0x64, 0x00, 0x8b, 0x64],
+    'Holiday': [0x01, 0x20, 0x0a, 0x55, 0x55, 0x60, 0x00, 0x00, 0x64, 0x00, 0xc2, 0x58, 0x01, 0x3e, 0x33, 0x00, 0xff, 0x46, 0x01, 0x1d, 0x64],
+    'Party': [0x01, 0x22, 0x04, 0x64, 0x64, 0x60, 0x00, 0x00, 0x64, 0x00, 0xd7, 0x5c, 0x00, 0xbc, 0x53, 0x00, 0x37, 0x1e, 0x00, 0x2c, 0x3f, 0x01, 0x61, 0x3f],
+    'Trend': [0x01, 0x23, 0x02, 0x64, 0x64, 0x60, 0x00, 0x00, 0x64, 0x01, 0x08, 0x4b, 0x00, 0xb1, 0x2f, 0x00, 0xcd, 0x57],
+    'Meditation': [0x01, 0x25, 0x03, 0x43, 0x43, 0x60, 0x00, 0x00, 0x64, 0x00, 0xb7, 0x35, 0x00, 0x9b, 0x54, 0x00, 0xcd, 0x61],
+    'Dating': [0x01, 0x26, 0x01, 0x59, 0x59, 0xe0, 0x00, 0x00, 0x64, 0x01, 0x19, 0x47, 0x01, 0x49, 0x3d, 0x00, 0xcd, 0x61, 0x00, 0x26, 0x64],
+    'Valentines Day': [0x01, 0x2a, 0x01, 0x64, 0x64, 0x60, 0x00, 0x00, 0x64, 0x01, 0x15, 0x64, 0x01, 0x05, 0x64, 0x01, 0x45, 0x64, 0x01, 0x2f, 0x64],
+    'Neon world': [0x01, 0x37, 0x0a, 0x5a, 0x5a, 0x60, 0x00, 0x00, 0x64, 0x00, 0x33, 0x58, 0x00, 0x18, 0x64, 0x01, 0x00, 0x45, 0x00, 0xe3, 0x5e, 0x00, 0xac, 0x30]
+} as const;
+
+const MUSIC_DATA = {
+    'Rock': [0x01, 0x01, 0x00, 0x03, 0x64, 0x32, 0x01, 0x00, 0x00, 0x64, 0x00, 0x00, 0x64, 0x00, 0x78, 0x64, 0x00, 0xf0, 0x64, 0x00, 0x3c, 0x64, 0x00, 0xb4, 0x64, 0x01, 0x2c, 0x64, 0x00, 0x00, 0x00],
+    'Jazz': [0x01, 0x01, 0x00, 0x02, 0x64, 0x32, 0x01, 0x00, 0x00, 0x64, 0x00, 0x00, 0x50, 0x00, 0x78, 0x50, 0x00, 0xf0, 0x50, 0x00, 0x3c, 0x50, 0x00, 0xb4, 0x50, 0x01, 0x2c, 0x50, 0x00, 0x00, 0x00],
+    'Classic': [0x01, 0x01, 0x00, 0x12, 0x64, 0x32, 0x01, 0x00, 0x00, 0x64, 0x00, 0x00, 0x64, 0x00, 0x78, 0x64, 0x00, 0xf0, 0x64, 0x00, 0x3c, 0x64, 0x00, 0xb4, 0x64, 0x01, 0x2c, 0x64, 0x00, 0x00, 0x00],
+    'Rolling': [0x01, 0x01, 0x01, 0x12, 0x64, 0x32, 0x01, 0x00, 0x00, 0x64, 0x00, 0x00, 0x64, 0x00, 0x78, 0x64, 0x00, 0xf0, 0x64, 0x00, 0x3c, 0x64, 0x00, 0xb4, 0x64, 0x01, 0x2c, 0x64, 0x00, 0x00, 0x00],
+    'Energy': [0x01, 0x01, 0x02, 0x12, 0x64, 0x32, 0x01, 0x00, 0x00, 0x64, 0x00, 0x00, 0x64, 0x00, 0x78, 0x64, 0x00, 0xf0, 0x64, 0x00, 0x3c, 0x64, 0x00, 0xb4, 0x64, 0x01, 0x2c, 0x64, 0x00, 0x00, 0x00],
+    'Spectrum': [0x01, 0x01, 0x03, 0x12, 0x64, 0x32, 0x01, 0x00, 0x00, 0x64, 0x00, 0x00, 0x64, 0x00, 0x78, 0x64, 0x00, 0xf0, 0x64, 0x00, 0x3c, 0x64, 0x00, 0xb4, 0x64, 0x01, 0x2c, 0x64, 0x00, 0x00, 0x00]
+} as const;
+
+function mapBrightnessToDp3(brightness: number): number {
+    const b = utils.toNumber(brightness, 'brightness');
+    const mapped = Math.round(utils.mapNumberRange(b, 0, 254, 10, 1000));
+    return Math.max(10, Math.min(1000, mapped));
+}
+
+const tzTuyaSpiConverters = {
```

Comment:
Can you convert these to value converters? You can find examples [here](https://github.com/Koenkk/zigbee-herdsman-converters/blob/b68e7f7a3d6407cde9856513ce702995fffe64a3/src/lib/tuya.ts#L601)